eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -2433,4 +2433,18 @@
     <Field name="detectionConfiguration" />
     <Bug pattern="UWF_UNWRITTEN_FIELD" />
   </Match>
+  
+  <!-- Disabling false positives in azure-core -->
+  <Match>
```

Comment:
Can we add a reason why these are false positives? A few months from now, if we decide to create another baseline for spotbugs, we won't remember why.